preset/src/shortcuts/alert.ts

```diff
@@ -0,0 +1,64 @@
+export const alertShortcuts: Record<string, string> = (() => {
+	const prefix = 'k-alert';
+
+	// base
+	const radius = '4px';
+	const basePx = '16px';
+	const basePy = '8px';
+
+	// icon
+	const iconMr = '8px';
+
+	// type
+	const typeCls = {};
+	const types = {
+		success: 'i-carbon-checkmark-filled',
+		error: 'i-carbon-error-filled',
+		warning: 'i-carbon-warning-filled',
+		info: 'i-carbon-information-filled'
+	};
+	Object.keys(types).forEach((type) => {
+		// type
+		typeCls[`${prefix}--${type}`] = `bg-ikun-${type}-100 text-ikun-${type}`;
+		// animate
+		typeCls[`${prefix}--${type}__animate`] = `animate-ikun-clicking--${type}`;
+		// icon
+		typeCls[`${prefix}--${type}__icon`] = types[type];
+		typeCls[`${prefix}--${type}__icon-color`] = `text-ikun-${type}`;
+		// close icon
+		typeCls[
+			`${prefix}--${type}__close-icon`
+		] = `hover:(bg-ikun-${type} [&>div.k-icon--base]:text-ikun-white b-rd-50%) active:(b-ikun-${type} k-btn-shadow--${type})`;
+	});
+
+	// title
+	const titleFontSize = '14px';
+
+	// description
+	const descriptionFontSize = '12px';
+	const descriptionMt = '8px';
+
+	// close
+	const closeTop = '10px';
+	const closeTopNoTitle = '8px';
+	const closeFontSize = '12px';
+
+	return {
+		// alert
```

Comment:
I prefer shortcut to use class objects to exhaustively list atomic class names, too abstract and not conducive to maintenance.
wdyt？

Reply:
> I prefer shortcut to use class objects to exhaustively list atomic class names, too abstract and not conducive to maintenance. wdyt？

This is just an attempt to reduce a lot of duplicate code. However, there are indeed difficult maintenance problems.